recipe/meta.yaml

```diff
@@ -1,29 +1,29 @@
-{% set version = "2.0.0" %}
+{% set version = "3.0.3" %}
 
 package:
   name: libjpeg-turbo
   version: {{ version }}
 
 source:
   url: https://github.com/libjpeg-turbo/libjpeg-turbo/archive/{{ version }}.tar.gz
-  sha256: f892fff427ab3adffc289363eac26d197ce3ccacefe5f5822377348a8166069b
-  # patches:
-  #   - 0001-Exclude-some-tests-when-not-WITH-12BIT.patch
+  sha256: a649205a90e39a548863a3614a9576a3fb4465f8e8e66d54999f127957c25b21
 
 build:
   number: 0
+  # We don't have yasm for s390x; libjpeg-turbo is needed for torchvision, which we won't build on s390x anyway
+  skip: True # [s390x]
   run_exports:
     - {{ pin_subpackage('libjpeg-turbo', max_pin='x') }}
 
 requirements:
   build:
     - {{ compiler('c') }}
     - cmake
-    - msinttypes  # [win and py<35]
-    - yasm
-    - jom  # [win]
-    - m2-patch  # [win]
-    - patch     # [not win]
+    # https://github.com/libjpeg-turbo/libjpeg-turbo/blob/7fa4b5b762c9a99b46b0b7838f5fd55071b92ea5/BUILDING.md?plain=1#L16
+    - yasm >=1.2.0
+    - ninja
+  run_constrained:
+    - jpeg <0.0.0a
 
 test:
```

Comment:
I think it is nice to test library files like `libjpeg.so.{version}`, l`ibjpeg.{version}.dylib`, etc. on all supported platforms. See https://github.com/libjpeg-turbo/libjpeg-turbo/blob/3.0.3/BUILDING.md#unx